### DISCUSSION TITLE
Record a Video

### DISCUSSION THREAD

**Discussion by antongenis** [bluenviron/mediamtx#545] [Questions]
Is there a possibility to record an incoming video? any config that can be changed?

**Comment by aler9** [comment]
Yes, follow instructions here
https://github.com/aler9/rtsp-simple-server#save-published-videos-to-disk

**Comment by antongenis** [comment]
Hi Alessandro,
I am actually trying to save a .mp4 format file. Is there a sample ffmpeg
command that rtsp-simple-server will accept?
Thank you


On Sun, Sep 5, 2021 at 7:12 PM Alessandro Ros ***@***.***>
wrote:

> Yes, follow instructions here
> https://github.com/aler9/rtsp-simple-server#save-published-videos-to-disk
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/aler9/rtsp-simple-server/discussions/545#discussioncomment-1282969>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAWQ43JH5TCT4ZVUYIXFTMLUAOJF3ANCNFSM5DBWID4A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Anton G

[a comment by an automated account is not shown]
